R/methods-epi_archive.R

```diff
@@ -833,22 +833,116 @@ epix_slide.epi_archive <- function(
     ref_time_values = NULL,
     new_col_name = NULL,
     all_versions = FALSE) {
-  # For an "ungrouped" slide, treat all rows as belonging to one big
-  # group (group by 0 vars), like `dplyr::summarize`, and let the
-  # resulting `grouped_epi_archive` handle the slide:
-  epix_slide(
-    group_by(x),
-    f,
-    ...,
-    before = before, ref_time_values = ref_time_values, new_col_name = new_col_name,
-    all_versions = all_versions
-  ) %>%
-    # We want a slide on ungrouped archives to output something
-    # ungrouped, rather than retaining the trivial (0-variable)
-    # grouping applied above. So we `ungroup()`. However, the current
-    # `dplyr` implementation automatically ignores/drops trivial
-    # groupings, so this is just a no-op for now.
-    ungroup()
+  ### START Copy pasta from grouped_epi_archive ###
```

Comment:
idea: also cross-ref in epix_slide.grouped_epi_archive if not already, to remind we need to update here?  Though it's easy to miss such comments in such a long function. 

Sorry, I've let this PR get out of sync with slide updates, so this copy-pasta would need updated as well, rather than along the way.  I was hoping we'd decide on whether to remove grouping capabilities first, but that got punted.  You feel strongly that the alternate code here is worth the copy-pasta?